roles/satellite-clone/vars/satellite_6.8.yml

```diff
@@ -14,3 +14,4 @@ db_packages:
   - rh-mongodb34-syspaths
   - rh-postgresql12-syspaths
   - rh-postgresql12-postgresql-evr
+satellite_ansible_version: 2.9
```

Comment:
While not hurting now, this is interpreted by YAML as a float, so if we ever have `3.0` here, it will fold it to `3`, which might be not present or mean something completely different. Better quote it.